checks/raw/fuzzing.go

```diff
@@ -105,14 +110,55 @@ var languageFuzzSpecs = map[clients.LanguageName]languageFuzzConfig{
 				"and test that specific properties are satisfied."),
 	},
 	clients.TypeScript: {
-		filePattern: "*.ts",
+		filePatterns: []string{"*.ts"},
 		// Look for direct imports of fast-check.
 		funcPattern: `(from\s+['"]fast-check['"]|require\(\s*['"]fast-check['"]\s*\))`,
 		Name:        fuzzerPropertyBasedTypeScript,
 		Desc: asPointer(
 			"Property-based testing in TypeScript generates test instances randomly or exhaustively " +
 				"and test that specific properties are satisfied."),
 	},
+	clients.Python: {
+		filePatterns: []string{"*.py"},
+		// Look for direct imports of fast-check.
+		funcPattern: `import atheris`,
```

Comment:
It's been a while since I've done python. How unique are these package names? Is this a good signature, in terms of false positives/negatives?

also all of these comments will need to be changed:
> // Look for direct imports of fast-check.

Reply:
I think it's a good signature. Going through https://github.com/search?q=import+atheris&type=code I only find fuzzing-relevant code using this.